fix(homepage-contributor-spotlight): preserve image aspect ratio

yashrajbharti opened the pull request:
<!-- 🙌 Thanks for contributing! Adding details below will help us to merge your PR faster. -->

### Description

<!-- ✍️ Summarize your changes in one or two sentences -->
Add object-fit: contain to .homepage-contributor-spotlight__logo img to prevent the 'MDN its the web' image from being stretched on Safari.

### Motivation

Help the Open source web!
<!-- ❓ Why are you making these changes and how do they help? -->

### Additional details

The fix removed the stretching on Safari
<img width="962" height="256" alt="Screenshot 2025-08-24 at 8 27 52 AM" src="https://github.com/user-attachments/assets/127e9c43-4933-44eb-afed-d79225e73af3" />

<!-- 🔗 Link to documentation, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

Fixes #613
<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->

caugner commented on components/homepage-contributor-spotlight/server.css:
```diff
@@ -79,6 +79,7 @@
     place-self: center;
     width: max-content;
     max-height: 8rem;
+    object-fit: contain;
```
To fix the stylelint issues:

```suggestion
    place-self: center;

    width: max-content;
    max-height: 8rem;

    object-fit: contain;
```

When committing locally, that should get fixed via lefthook (using the pre-commit hook).

caugner commented:
> @yashrajbharti We currently require commit signatures, and recommend [reusing your SSH key](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key) to sign your commits. Once set up, you should be able to resign your commits with an interactive rebase:
> 
> ```
> git fetch origin main
> git rebase --exec 'git commit --amend --no-edit -n -S' -i origin/main
> ```

@yashrajbharti Did these step work for you? Afterwards, you will need to `git push --force-with-lease` once, for your signed commits to appear here.

yashrajbharti commented:
@caugner yes I am able to!, this should work!

Commit:
Fix homepage contributor spotlight image aspect ratio

Add object-fit: contain to .homepage-contributor-spotlight__logo img
to prevent the 'MDN its the web' image from being stretched on Safari.

Fixes #613
